commands/verify/verify.go

```diff
@@ -96,44 +154,75 @@ func DoVerifyAccordingConfig() error {
 	e2eConfig := config.GlobalConfig.E2EConfig
 
 	retryCount := e2eConfig.Verify.RetryStrategy.Count
-	if retryCount <= 0 {
-		retryCount = 1
-	}
+	retryCount = checkForRetryCount(retryCount)
+
 	interval, err := parseInterval(e2eConfig.Verify.RetryStrategy.Interval)
 	if err != nil {
 		return err
 	}
 
 	failFast := e2eConfig.Verify.FailFast
+	concurrency := e2eConfig.Verify.Concurrency
+
+	var errCollection multierror.Error
+
+	errContain := ErrContain{
+		errAddr: &errCollection,
+	}
 
-	var errCollection *multierror.Error
-	for idx, v := range e2eConfig.Verify.Cases {
-		if v.GetExpected() == "" {
-			errMsg := fmt.Sprintf("the expected data file for case[%v] is not specified\n", idx)
-			if failFast {
-				return errors.New(errMsg)
+	if concurrency {
+		var waitGroup sync.WaitGroup
+		VerifyInfo := verifyInfo{
+			retryCount,
+			interval,
+			failFast,
+		}
+		chanBool := make(chan bool, 1)
```

Comment:
The name `chanBool` is not informative, `stopChan` may be better. Besides, we can use `chan struct{}`, you can search related articles to learn about it.